src/background.ts

```diff
@@ -1336,18 +1364,38 @@ ipcMainHandle("THEME", (_, { newData }) => {
   if (newData !== undefined) {
     store.set("currentTheme", newData);
     return;
+  } else {
+    return getThemes();
   }
+});
+
+const getThemes = () => {
+  const currentTheme = store.get("currentTheme");
   const dir = path.join(__static, "themes");
-  const themes: ThemeConf[] = [];
   const files = fs.readdirSync(dir);
+  const themes: ThemeConf[] = [];
   files.forEach((file) => {
-    const theme = JSON.parse(fs.readFileSync(path.join(dir, file)).toString());
+    const theme = JSON.parse(
+      fs.readFileSync(path.join(dir, file)).toString()
+    ) as ThemeConf;
     themes.push(theme);
   });
-  return { currentTheme: store.get("currentTheme"), availableThemes: themes };
-});
+  return { currentTheme: currentTheme, availableThemes: themes };
+};
+
+const getCurrentTheme = (): ThemeConf | null => {
+  const { currentTheme, availableThemes } = getThemes();
+  let currentThemeRef = null;
+  availableThemes.forEach((theme) => {
+    if (theme.name == currentTheme) {
+      currentThemeRef = theme;
+    }
+  });
+  return currentThemeRef;
+};
```

Comment:
I think it's better to `throw` the bug when unexpected behavior comes in, so that the cause of the bug is easier to find!
(I think this is not much of a problem as it is)

```suggestion
const getCurrentTheme = (): ThemeConf => {
  const { currentTheme, availableThemes } = getThemes();
  let currentThemeRef = null;
  availableThemes.forEach((theme) => {
    if (theme.name == currentTheme) {
      currentThemeRef = theme;
    }
  });
  if ( currentThemeRef != null ) throw new Error("currentTheme is null");
  return currentThemeRef;
};
```

想定外の挙動が来たときは`throw`したほうがバグの原因がわかりやすくて良いと思います！
（そのままでもあまり問題ではないと思います）